src/assets/css/login-prompt.css

```diff
@@ -0,0 +1,94 @@
+@font-face {
+  font-family: 'ProximaSoft';
+  font-style: normal;
+  font-weight: normal;
+  src: url('/fonts/ProximaSoft/3955CC_4_0.woff2') format('woff2'),
+    url('/fonts/ProximaSoft/3955CC_4_0.woff') format('woff'),
+    url('/fonts/ProximaSoft/3955CC_4_0.ttf') format('truetype');
+}
+
+body {
+  font-family: 'ProximaSoft', 'Droid Sans', 'helvetica neue', 'arial', sans-serif;
+  font-size: 14px;
+  line-height: 20px;
+  margin: 0;
+}
+
+.button--green {
+  background-color: #00bf68;
+  border: 1px solid #00bf68;
+  border-radius: 4px;
+  color: #fff;
+  cursor: pointer;
+  display: inline-block;
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 1;
+  overflow: hidden;
+  padding: 8px 16px;
+  position: relative;
+  text-decoration: none;
+  text-overflow: ellipsis;
+  transition: background-color 0.15s;
+  transition-timing-function: linear;
+  vertical-align: middle;
+  white-space: nowrap;
+}
+
+.button--green:hover {
+  background-color: #24de7f;
+}
+
+.button--green:active,
+.button--green:focus {
+  background-color: #138f51;
+}
+
+
+.svg-inline--fa {
+  display: inline-block;
+  height: 1em;
+  vertical-align: middle;
+}
+
+.login-prompt__roundingwell-logo {
+  margin-bottom: 50px;
```

Comment:
Ah so now all px settings everywhere should be multiples of 4px and ideally multiples of 8px